rust/gui-client/src/settings.ts

```diff
@@ -154,7 +154,7 @@ async function clearLogs() {
       console.error(e);
     })
     .finally(() => {
-      logCountOutput.innerText = "0 files, 0 MB";
+      countLogs();
```

Comment:
If the permissions are wrong, the GUI may fail to clear all the logs. In that case, I want the display to be accurate.